@xen-orchestra/web-core/lib/components/ui/label/UiLabel.vue

```diff
@@ -42,17 +41,10 @@ defineProps<{
     }
   }
 
-  .link {
-    display: flex;
-    align-items: center;
-    gap: 0.8rem;
+  .learn-more-link {
```

Comment:
Maybe we could let `.link` for more clarity ? it remains in the naming logic as with `.icon`

Reply:
The problem here is that the `.link` class is also a utility that applies styles we don't want here, so that is why I renamed it.

Reply:
I'm wondering, now that we have the `UiLink` component, could we perhaps update the components using the `.link` class and remove it from the utilities when no longer needed?
So we could use `.link` safely, without side effects.
Any thoughts on this @ByScripts?

Reply:
Yes, the global `.link` class should be removed and be replaced with `UiLink` where needed.